pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -1998,6 +2044,17 @@
           %leave  ~
       ==
     --
+  ::  +set-response: remember (or update) a cache mapping
+  ::
+  ++  set-response
+    |=  [url=@t entry=(unit cache-entry)]
+    ^-  [(list move) server-state]
+    =/  aeon
+      =/  cache-val  (~(get by cache.state) url)
+      ?~(cache-val 0 +(aeon.u.cache-val))
```

Comment:
Let's start the aeon at 1, so that aeon 0 can mean there's no mapping yet.  This might be useful if, for example, someone scrys for the latest aeon on a path later.  Clay starts at 1 for this reason.